include/SH3/system/glprogram.hpp

```diff
@@ -26,6 +27,7 @@ namespace sh3_graphics
     public:
 
         sh3_glprogram(const std::string& name) : programName(name){Load(name);};
+        sh3_glprogram(const std::string& name, std::vector<std::string> const &attribs) : programName(name){Load(name, attribs);};
```

Comment:
Perhaps not, I was doing it to keep it consistent with other modules and to add a way to attempt to "reload" if something fails. I'll move the code for `Load` into the constructor and move the constructor into `glprogram.cpp`